### PR TITLE
remove secure storage

### PR DESCRIPTION
**Pull request by ciaranRoche**
## Motivation

Based on feedback, we have removed the secure storage page and replaced it with an under construction page.

This change also makes changes to allow all under construction pages to use the same single html page.

We also have updated the hex color values to match that of the ios and android app

## Screenshot

under-cons-2

### CODE REVIEW

File: src/pages/storage/storage.html [hunk 1]
```diff
@@ -1,25 +1,33 @@
...
+          <p>This feature is currently under construction for this platform.</p>
+          <p>If you would like to get involved in this work, please check out our community contribution page.</p>
+          <p>
+            <a href="#" onclick="window.open('https://aerogear.org/community/#guides', '_system');">aerogear.org/community/#guides</a>
+          </p>
+        </div>
+      </ion-label>
+    </ion-col>
+  </ion-row>
+</ion-grid>
```
Comment (wei-lee) [inline src/pages/storage/storage.html]: so for each of the `undersconstruction page`, we have the same html code in a few different pages?

How hard would be to just have one place for the "underconstruction" html page, and reference it in different pages?

**Comment by ciaranRoche** [comment]
@wei-lee I have made that update to use a single construct page, good catch

File: src/pages/certPinning/certPinning.ts [hunk 2]
```diff
@@ -5,12 +5,14 @@ import { ToastController } from 'ionic-angular';
 
 @Component({
     selector: 'page-certPinning',
-    templateUrl: 'certPinning.html'
```
Comment (pawelpaszki) [inline src/pages/certPinning/certPinning.ts]: @ciaranRoche now you have a number of unused html templates. We might want to remove them or create a fully-fledged underConstruction page with selector and place it directly into the html templates (if we want to leave them), by providing the selector in the html templates:

<page-underConstruction></page-underConstruction>

**Comment by ciaranRoche** [comment]
@pawelpaszki mind having another look, good catch with that :+1:

File: src/pages/sso/sso.ts [hunk 3]
```diff
@@ -5,12 +5,14 @@ import { ToastController } from 'ionic-angular';
 
 @Component({
     selector: 'page-sso',
-    templateUrl: 'sso.html'
+    templateUrl: '../underConstruction/construction.html'
```
Comment (pawelpaszki) [inline src/pages/sso/sso.ts]: @ciaranRoche is this the correct template?

Comment (ciaranRoche) [reply src/pages/sso/sso.ts]: nice spot :+1:

**Comment by pawelpaszki** [comment]
@ciaranRoche looks good to merge after the last update

**Pull request merged**